config/typescript/modern.json

```diff
@@ -19,7 +19,8 @@
 			"DOM",
 			"webWorker"
 		],
-		"moduleResolution": "node",
+		"module": "node16",
+		"moduleResolution": "node16",
```

Comment:
Would it make more sense to use nodenext instead of node16 as we need to move to a higher node version in the future? Or will making this nodenext start causing things to fail?

Reply:
Since this is TSC options, this affects the distributable. If we move this to node-next, then we potentially lose support for consumers using node16 (as TSC will not build with node16 in mind for the distributable artifact).

We ultimately want to move to node-next (which is ultimately LTS) and require all consumers to be on node LTS (which is now node 20), but this is not the case from the Webex Web Client's side.

Reply:
Sounds good. Let's do along with the move to node 20.